### DISCUSSION TITLE
It's better each issues/milestones not be linked to a specific repository

### DISCUSSION THREAD

**Discussion by sundaycrafts** [community/community#9364] [Projects and Issues]
First of all, thank you for the opportunity to try the new project feature.

In my situation, every issue or milestone of some project is tied to the business value, not specific implementation (= a specific repository).

So it's awkward they are always linked to the specific repository. They may have relevant to a repository but not always.

And normally we have tons of issues. So we need more keyboard shortcuts to easy to fly!

**Comment by FrameMuse** [comment]
Absolutely agree with that. We have a lot of `shared` issues and I'd like to see separate labels for projects. Also, you could add issues to all chosen repositories so they can be closed by a commit and display them as a checklist in the project issue.

**Comment by anders-jagard** [comment]
In my case we have different teams collaborating towards same Milestone with different products and therefor different repos. In this case I would like to have possibilty to create a Milestone that can include/have issues from different repositories.

From a Product owner perspective it would be nice to visualize this in a project view for both developers and stakeholders...
